packages/perseus/src/widgets/interactive-graphs/graphs/circle.tsx

```diff
@@ -51,40 +52,25 @@ function CircleGraph(props: CircleGraphProps) {
     const outerPointsId = id + "-outer-points";
 
     // Aria label strings
-    const circleGraphAriaLabel = strings.srCircleGraph;
-    const circleShapeAriaLabel = strings.srCircleShape({
-        centerX: srFormatNumber(center[0], locale),
-        centerY: srFormatNumber(center[1], locale),
-    });
-    const circleRadiusPointAriaLabel = strings.srCircleRadiusPoint({
-        radiusPointX: srFormatNumber(radiusPoint[0], locale),
-        radiusPointY: srFormatNumber(radiusPoint[1], locale),
-    });
-    const circleRadiusDescription = strings.srCircleRadius({
-        radius,
-    });
-    const circleOuterPointsDescription = strings.srCircleOuterPoints({
-        point1X: srFormatNumber(center[0] + radius, locale),
-        point1Y: srFormatNumber(center[1], locale),
-        point2X: srFormatNumber(center[0], locale),
-        point2Y: srFormatNumber(center[1] + radius, locale),
-        point3X: srFormatNumber(center[0] - radius, locale),
-        point3Y: srFormatNumber(center[1], locale),
-        point4X: srFormatNumber(center[0], locale),
-        point4Y: srFormatNumber(center[1] - radius, locale),
-    });
+    const {
+        srCircleGraph,
+        srCircleShape,
+        srCircleRadiusPoint,
+        srCircleRadius,
+        srCircleOuterPoints,
+    } = describeCircleGraph(graphState, {strings, locale});
```

Comment:
I like this way of splitting up the logic! Makes it much easier to read and test :)